recipe/install-gcc.sh

```diff
@@ -215,26 +154,31 @@ pushd ${PREFIX}
       *script*executable*)
       ;;
       *executable*)
-        ${SRC_DIR}/gcc_built/bin/${CHOST}-strip --strip-all -v "${_file}" || :
+        ${BUILD_PREFIX}/bin/${CHOST}-strip --strip-all -v "${_file}" || :
       ;;
     esac
   done
 popd
 
 #${PREFIX}/bin/${CHOST}-gcc "${RECIPE_DIR}"/c11threads.c -std=c11
 
+mkdir -p ${PREFIX}/${CHOST}/lib
+
 if [[ "$target_platform" == "$ctng_target_platform" ]]; then
   # making these this way so conda build doesn't muck with them
-  pushd ${PREFIX}/${CHOST}/sysroot/lib
```

Comment:
I don't understand this change in layout. Is this going to affect builds with CDTs?

Reply:
No, this is not going to affect CDTs. What this change does is, it avoids mucking with the sysroot with files from the compiler, so that only `sysroot_{{ target_platform }}` and CDTs add files to them. One advantage will be that we'll be able to swap out the conda sysroot with system one by doing
```bash
rm -rf ${PREFIX}/${CHOST}/sysroot
ln -sf / ${PREFIX}/${CHOST}/sysroot
```

Reply:
Cool. Thanks for the explanation!